dashboard/src/components/Catalog/Catalog.v2.tsx

```diff
@@ -0,0 +1,152 @@
+import { RouterAction } from "connected-react-router";
+import { uniq } from "lodash";
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import FilterGroup from "components/FilterGroup/FilterGroup";
+import Alert from "components/js/Alert";
+import Column from "components/js/Column";
+import Row from "components/js/Row";
+import { IFeatureFlags } from "shared/Config";
+import { IChartState, IClusterServiceVersion } from "../../shared/types";
+import { escapeRegExp } from "../../shared/utils";
+import CardGrid from "../Card/CardGrid.v2";
+import LoadingWrapper from "../LoadingWrapper/LoadingWrapper.v2";
+import PageHeader from "../PageHeader/PageHeader.v2";
+import SearchFilter from "../SearchFilter/SearchFilter.v2";
+
+import { app } from "shared/url";
+import "./Catalog.v2.css";
+import CatalogItems from "./CatalogItems";
+
+interface ICatalogProps {
+  charts: IChartState;
+  repo: string;
+  filter: string;
+  fetchCharts: (namespace: string, repo: string) => void;
+  pushSearchFilter: (filter: string) => RouterAction;
+  namespace: string;
+  kubeappsNamespace: string;
+  getCSVs: (namespace: string) => void;
+  csvs: IClusterServiceVersion[];
+  featureFlags: IFeatureFlags;
+}
+
+function Catalog(props: ICatalogProps) {
+  const {
+    charts: {
+      isFetching,
+      selected: { error },
+      items: charts,
+    },
+    fetchCharts,
+    namespace,
+    pushSearchFilter,
+    getCSVs,
+    csvs,
+    repo,
+    filter: propsFilter,
+  } = props;
+  const [searchFilter, setSearchFilter] = useState(propsFilter);
+  const [typeFilter, setTypeFilter] = useState([] as string[]);
```

Comment:
For the type filter, I'd use a `radio` button with the options:

- All
- Charts
- Operators

It's not common to display a checkbox already marked, but the default behavior should be the "All" one. Also, conceptually it's easier to identify you click on an option to include only those items than excluding the group.

Reply:
mmh, for consistency with the rest of filters, I would use checkboxes. My idea is to have the typical filter checkbox list in which marking none of them is the same result than marking all (not sure if my screenshot confused you). An example from amazon:

![Screenshot from 2020-07-15 17-42-05](https://user-images.githubusercontent.com/4025665/87566029-b6bcc100-c6c2-11ea-928f-1924b6603537.png)

Marking none of the items show all of them, click in one filter only those elements.

Reply:
Okey! We can left it based on checkboxes. Since they're not active by default, I think it's fine :)